sahara.c

```diff
@@ -345,7 +345,10 @@ static ssize_t sahara_debug64_one(struct qdl_device *qdl,
 				goto out;
 			}
 
-			write(fd, buf, n);
+			if (n != write(fd, buf, n)) {
```

Comment:
I think it should be slightly different:
- error if returned value is less or equal to zero
- increase offset by the return value otherwise

Reply:
thanks, fixed!